api/metric/api.go

```diff
@@ -31,9 +31,16 @@ type Provider interface {
 	Meter(name string) Meter
 }
 
-// LabelSet is an implementation-level interface that represents a
-// []core.KeyValue for use as pre-defined labels in the metrics API.
-type LabelSet interface {
+// Labels is a convenience method to build a []core.KeyValue{}.  For example,
```

Comment:
Seems a bit trivial, but I'm not strongly opinionated here.

Reply:
I took it out.